core-authentication/src/test/java/gg/solarmc/loader/authentication/test/Player.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * DataLoader
+ * Copyright © 2021 SolarMC Developers
+ *
+ * DataLoader is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * DataLoader is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with DataLoader. If not, see <https://www.gnu.org/licenses/>
+ * and navigate to version 3 of the GNU Affero General Public License.
+ */
+
+package gg.solarmc.loader.authentication.test;
+
+import gg.solarmc.loader.Transaction;
+import gg.solarmc.loader.authentication.UserWithDataNotYetLoaded;
+import gg.solarmc.loader.authentication.User;
+
+import java.util.UUID;
+
+public final class Player implements UserWithDataNotYetLoaded {
```

Comment:
Minor nitpicky suggestion - Change this to DataPlayer or something so we don't get multiple classes with the same name?

Reply:
As this is a test class, we will not have multiple classes on the module path of core-authentication testing named Player.